environments/example-credentials/services/fleetms/overlays/appsody-env-patch.yaml

```diff
@@ -1,8 +1,16 @@
 - op: add
   path: "/spec/env/-"
   value:
-    name: KAFKA_APIKEY
+    name: KAFKA_USER
     valueFrom:
       secretKeyRef:
-        name: "eventstreams-apikey"
-        key: binding
\ No newline at end of file
+        name: "eventstreams-cred"
```

Comment:
Based on this change to `eventstreams-cred` from `eventstreams-apikey`, you're going to need to change the SecretGenerators (via https://github.com/jesusmah/refarch-kc-gitops/blob/scram/environments/example-credentials/env/base/kustomization.yaml#L10-L13) to create new secrets with the new name.

The current Secrets being generated were expected to be `eventstreams-apikey` but you've updated them to be more generically named `eventstreams-cred` (rightly so - due to now including username and password instead of just password). We will need to align this for all items in the PR. 

Anywhere there is `eventstreams-apikey` under `environments`, we should be able to replace it with `eventstreams-cred`.

Reply:
I believe I have addressed the above in my latest commit